packages/mobile/src/screens/chat-screen/ChatUserListItem.tsx

```diff
@@ -0,0 +1,165 @@
+import type { User } from '@audius/common'
+import { Text, View, TouchableOpacity } from 'react-native'
+import { useDispatch, useSelector } from 'react-redux'
+
+import IconUser from 'app/assets/images/iconUser.svg'
+import LoadingSpinner from 'app/components/loading-spinner'
+import { ProfilePicture } from 'app/components/user'
+import { UserBadges } from 'app/components/user-badges'
+import { makeStyles } from 'app/styles'
+import { useThemeColors } from 'app/utils/theme'
+
+const { createChat } = chatActions
+const { getCanMessageUser } = chatSelectors
+
+const messages = {
+  followsYou: 'Follows You',
+  followers: 'Followers',
+  cannotMessage: 'Cannot Be Messaged'
+}
+
+const useStyles = makeStyles(({ spacing, palette, typography }) => ({
+  rootContainer: {
+    backgroundColor: palette.white,
+    flexGrow: 1
+  },
+  profilePicture: {
+    height: spacing(18),
+    width: spacing(18)
+  },
+  border: {
+    borderBottomColor: palette.neutralLight4,
+    borderBottomWidth: 1
+  },
+  userContainer: {
+    display: 'flex',
+    flexDirection: 'row',
+    alignItems: 'center',
+    padding: spacing(4)
+  },
+  userNameContainer: {
+    flexGrow: 1,
+    display: 'flex',
+    flexDirection: 'column',
+    marginLeft: spacing(2.5),
+    gap: spacing(1)
+  },
+  userName: {
+    fontSize: typography.fontSize.small,
+    fontWeight: 'bold',
+    color: palette.neutral
+  },
+  followContainer: {
+    marginTop: spacing(1),
+    display: 'flex',
+    flexDirection: 'row',
+    alignItems: 'center',
+    justifyContent: 'space-between'
+  },
+  handle: {
+    fontSize: typography.fontSize.small,
+    color: palette.neutral
+  },
+  followersContainer: {
+    display: 'flex',
+    flexDirection: 'row',
+    alignItems: 'center'
+  },
+  followersCount: {
+    fontWeight: '800',
+    marginHorizontal: spacing(1),
+    color: palette.neutralLight4,
+    fontSize: typography.fontSize.small
+  },
+  followers: {
+    color: palette.neutralLight4,
+    fontSize: typography.fontSize.small
+  },
+  iconUser: {
+    height: spacing(4),
+    width: spacing(4)
+  },
+  followsYouTag: {
+    fontSize: typography.fontSize.xxs,
+    fontFamily: typography.fontByWeight.heavy,
+    letterSpacing: 0.64,
+    textTransform: 'uppercase',
+    color: palette.neutralLight4,
+    borderWidth: 1,
+    borderRadius: spacing(1),
+    borderColor: palette.neutralLight4,
+    paddingVertical: spacing(1),
+    paddingHorizontal: spacing(2)
+  },
+  dim: {
+    opacity: 0.5
+  }
+}))
+
+type ChatUserListItemProps = {
+  user: User
+}
+
+export const ChatUserListItem = ({ user }: ChatUserListItemProps) => {
+  const styles = useStyles()
+  const palette = useThemeColors()
+  const dispatch = useDispatch()
+  const canMessageUser = useSelector((state) =>
+    getCanMessageUser(state, user.user_id)
+  )
+
+  const handlePress = useCallback(
+    (user) => {
+      dispatch(createChat({ userIds: [user.user_id] }))
+    },
+    [dispatch]
+  )
+
+  if (!user) {
```

Comment:
* Would love to define things like this as explicit loading states since `props.user` could be technically be falsey for other reasons. 
* Is there anything else we need to wait for to ensure this component works correctly
* You could move this logic up a level so that this component is guaranteed to be rendered with a `User`. If I understand the code correctly, we're using this to render a spinner while we load results? That logic probably belongs in the parent.

Reply:
Good call. Actually we already ensure the user exists in the parent so didn't need this check anyway. Yay!

Reply:
Great call. Actually the data we pass in from the parent is from a selector, so we don't need this check anyway. Yay!